sig-auth/README.md

```diff
@@ -37,6 +37,39 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-auth)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fauth)
 
+## Subprojects
+
+The following subprojects are owned by sig-auth:
+- **audit-logging**
+  - Owners:
+    - Tim Allclair
+- **authenticators**
```

Comment:
Should pod identity be split out from this? Mostly just because it deals with a lot more than just the authenticator interface plugin. I suppose you could make the same argument for a lot of other pieces though.